graphsdk/src/androidTest/java/com/microsoft/graph/serializer/CalendarSerializerTests.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.graph.serializer;
+
+import android.test.AndroidTestCase;
+
+import java.util.Calendar;
+
+
+/**
+ * Test cases for {@see CalendarSerializer}
+ */
+public class CalendarSerializerTests extends AndroidTestCase {
+
+    public void testCalendarSerializer() throws Exception {
```

Comment:
Is this needed to get more code coverage, or do we just need to expand on the ISO8601Test cases?